person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative('./nameable')
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :nameable
 
   def initialize(age, nameable, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Its a requirement that @rentals should be initialized in the `Person` class. Kindly do so, here is an example
e.g

```
 def initialize(age, name = "Unknown", parent_permission = true)
  
    @rentals = []
  end
```